graphjet-demo/src/main/java/com/twitter/graphjet/demo/PageRankGraphJetDemo.java

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2016 Twitter. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twitter.graphjet.demo;
+
+import com.twitter.graphjet.algorithms.PageRank;
+import com.twitter.graphjet.bipartite.segment.IdentityEdgeTypeMask;
+import com.twitter.graphjet.directed.OutIndexedPowerLawMultiSegmentDirectedGraph;
+import com.twitter.graphjet.stats.NullStatsReceiver;
+import it.unimi.dsi.fastutil.longs.LongOpenHashSet;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+import org.kohsuke.args4j.ParserProperties;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Simple benchmark program that loads a graph and runs PageRank over it.
+ */
+public class PageRankGraphJetDemo {
+  private static class PageRankGraphJetDemoArgs {
+    @Option(name = "-inputFile", metaVar = "[value]",
+        usage = "input data", required = true)
+    String inputFile;
+
+    @Option(name = "-maxSegments", metaVar = "[value]",
+        usage = "maximum number of segments")
+    int maxSegments = 20;
```

Comment:
Yes, once I get everything merged to master I will start playing with the parameters.